docs/earth-config/earth-config.md

```diff
@@ -0,0 +1,34 @@
+# Earthly config file
+
+Global configuration values for earth can be stored on disk in the configuration file.
+
+By default, earth reads the configuration file `~/.earthly/config.yaml`; however, it can also be
+overridden with the `--config` command flag option.
+
+Currently only https authentication configuration is supported by the config file.
+
+## Format
+
+The earthly config file is a [yaml](https://yaml.org/) formatted file that looks like:
+
+```yaml
+git:
+    <site>:
+        auth: https
+        user: <username>
+        password: <password>
+    <site2>:
+        ...
+```
+
+### HTTPS Authenticating
+
+To configure https authentication, the auth mode must be set to https
+in addition to the user and password values.
+
+### SSH Authenticating
+
+By default earth uses ssh authentication. It can be explicitly set by setting `auth: ssh`,
+or implicitly set by omitting the site-specific configuration.
+
+See the [Authenticat guide](guides/auth.md) for more details.
```

Comment:
It feels like this is more like a guide rather than reference. I would keep the auth guide page to serve as the guide for these settings.

For this particular page, I would just keep it purely a reference page, with just format and reference. See the [docker-compose reference](https://docs.docker.com/compose/compose-file/) as an example.

Perhaps this could be the structure, roughly:

```markdown
## Format
...
## Global configuration reference (future)
...
### some-config-item
...
### another-config-item
...
## Git configuration reference
...
### auth
...
### user
...
### password
...
```

Reply:
thanks for the example, I've fixed this up.